Phonebook.Frontend/src/app/modules/table/components/table/table.component.html

```diff
@@ -277,15 +274,33 @@
   </mat-table>
 </div>
 <mat-card *ngIf="!dataSource.data.length" class="notfound-container">
+  <ng-container class="spinner-container">
+    <mat-spinner *ngIf="dataSource.loading$ | async" diameter="30"></mat-spinner
+  ></ng-container>
   <ng-container
+    *ngIf="this.loaded === true && this.error === false && displayedColumns.length !== 0"
```

Comment:
If `loading$` is false the data has been loaded.

Reply:
I know that that should work in theory but dataSource.loading$ === false didn't work for me. It simply didn't display the message after the data had been loaded. I searched for something that didn't exist and the search result was zero but the screen was empty...

Reply:
That's because `loading$` is an Observable, you would have to use the `| async` pipe on it. 
I updated the branch because I tried out if the errors are handled and stumbled upon a bug @paule96 produced and I let through. It should be working as expected now.